skl2onnx/operator_converters/powertransformer.py

```diff
@@ -0,0 +1,131 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import numpy
+
+from onnxconverter_common import apply_identity
+from onnx import TensorProto
+
+from ..common._registration import register_converter
+from ..algebra.onnx_ops import (
+    OnnxAdd, OnnxSub, OnnxPow, OnnxDiv, OnnxMul,
+    OnnxCast, OnnxNot, OnnxLess, OnnxLog, OnnxNeg, OnnxImputer
+)
+
+
+def convert_powertransformer(scope, operator, container):
+    """Converter for PowerTransformer"""
+    op_in = operator.inputs[0]
+    op_out = operator.outputs[0].full_name
+    op = operator.raw_operator
+    lambdas = op.lambdas_
+
+    # tensors of units and zeros
+    ones_ = OnnxDiv(op_in, op_in)
```

Comment:
You need to specify op_version=opv where opv = container.target_opset (see https://github.com/onnx/sklearn-onnx/blob/master/skl2onnx/operator_converters/k_means.py#L79). Otherwise, the converter only picks the latest version of every operator.

Reply:
Ok, Thank you!